decidim-core/app/packs/src/decidim/redesigned_index.js

```diff
@@ -110,7 +110,7 @@ const initializer = (element = document) => {
   window.theDataPicker = new DataPicker($(".data-picker"));
   window.focusGuard = new FocusGuard(element.querySelector("body"));
 
-  $(element).foundation();
+  // $(element).foundation();
```

Comment:
@Crashillo sure about this change?

Reply:
Initially, I comment that out due to `[data-tooltip]` attribute could interfere between foundation and the new tooltips. But this could cause that a lot of tests will fail, so I've brought that back, since I've tested there's no intrference